testutils/memory.go

```diff
@@ -0,0 +1,120 @@
+package testutils
+
+import (
+	"github.com/src-d/metadata-retrieval/github/graphql"
+	"gopkg.in/src-d/go-log.v1"
+)
+
+// Memory implements the storer interface
+type Memory struct {
+	Organization *graphql.Organization
+	Repository   *graphql.RepositoryFields
+	Topics       []string
+	Users        []*graphql.UserExtended
+	Prs          []*graphql.PullRequest
+	PrComments   []*graphql.IssueComment
+}
+
+// SaveOrganization stores an organization in memory,
+// it also initializes the list of users
+func (s *Memory) SaveOrganization(organization *graphql.Organization) error {
+	log.Infof("organization data fetched for %s\n", organization.Login)
+	s.Organization = organization
+	// Initialize users to 0 for each repo
+	s.Users = make([]*graphql.UserExtended, 0)
+	return nil
+}
+
+// SaveUser appends a user to the user list in memory
+func (s *Memory) SaveUser(user *graphql.UserExtended) error {
+	log.Infof("user data fetched for %s\n", user.Login)
+	s.Users = append(s.Users, user)
+	return nil
+}
+
+// SaveRepository stores a repository and its topics in memory and
+// initializes PRs and PR comments
+func (s *Memory) SaveRepository(repository *graphql.RepositoryFields, topics []string) error {
+	log.Infof("repository data fetched for %s/%s\n", repository.Owner.Login, repository.Name)
+	s.Repository = repository
+	s.Topics = topics
+	// Initialize prs and comments to 0 for each repo
+	s.Prs = make([]*graphql.PullRequest, 0)
+	s.PrComments = make([]*graphql.IssueComment, 0)
+	return nil
+}
+
+// SaveIssue TODO(Kyriakos)
```

Comment:
please consider this guide on TODOs https://github.com/src-d/guide/blob/806661020bf47a2a736e7b72aee1095d42a41774/engineering/conventions.md#todo-and-fixme-comments